src/KingSystem/Resource/resResourceDamageParam.h

```diff
@@ -0,0 +1,155 @@
+#include <agl/Utils/aglParameter.h>
+#include <agl/Utils/aglParameterObj.h>
+#include <container/seadBuffer.h>
+#include <prim/seadEnum.h>
+#include "KingSystem/Resource/resResource.h"
+#include "KingSystem/Utils/ParamIO.h"
+
+namespace ksys::res {
+
+class DamageParam : ParamIO, Resource {
+    SEAD_RTTI_OVERRIDE(DamageParam, Resource)
+
+public:
+    DamageParam();
+    ~DamageParam() = default;
+
+    bool needsParse() const override { return true; }
+    bool ParamIO_m0() override { return false; }
+
+    f32 getDamageRate(sead::SafeString& damageSource);
+    const sead::SafeString& getDamageSize(sead::SafeString& damageSource,
```

Comment:
```suggestion
    const sead::SafeString& getDamageReaction(const sead::SafeString& damage_source,
```

This seems to return the reaction